doc/manual/utils.nix

```diff
@@ -56,6 +56,21 @@ rec {
       body = ''
           ${description}
 
+        '' + (if experimental-feature != null then ''
+          > **Warning**
+          > This setting is part of an experimental feature.
+
+          To change this setting, you need to make sure the corresponding experimental feature,
+          [`${experimental-feature}`](@docroot@/contributing/experimental-features.md#xp-feature-${experimental-feature}),
+          is enabled.
+          For example, include the following in [`nix.conf`](#):
```

Comment:
We should explain when to use it and when not to use it, probably by linking the a page about experimental features?

Reply:
The section on the specific experimental feature is the same page, but also I made the warning above link the page as a whole (not URL fragment).